### PR TITLE
Re-add dash-phylogeny app

### PR DESCRIPTION
**Pull request by ycaokris**
Issue for app: #https://github.com/plotly/dash-sample-apps/issues/26

# App pull request
- [ ] This is a new app
- [x] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version): https://dash-playground.plotly.host/dash-phylogeny/
- Current gallery app URL: https://dash-gallery.plotly.host/dash-phylogeny/

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [x] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [x] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [x] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [x] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

**Comment by ycaokris** [comment]
- [x] Upgraded Dash Version
- [x] Restyled and mobile responsiveness
- [x] Deployed on test server

File: apps/dash-phylogeny/app.py [hunk 1]
```diff
@@ -0,0 +1,811 @@
+# -*- coding: utf-8 -*-
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objs as go
```
Comment (sstripps1) [inline apps/dash-phylogeny/app.py]: Is this import used anywhere else in the app?

File: apps/dash-phylogeny/utils.py [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+    elif max_date - min_date > 50:
+        step = 10
+    marks_data = []
+    for i in range(int(min_date), int(max_date + 1), step):
+        marks_data.append(str(i))
+
+    if i < int(max_date):
+        marks_data.append(str(max_date))
+
+    year = marks_data
```
Comment (sstripps1) [inline apps/dash-phylogeny/utils.py]: It doesn't look like this variable is needed for the rest of the function

File: apps/dash-phylogeny/utils.py [hunk 3]
```diff
@@ -0,0 +1,743 @@
...
+    text = []
+
+    for cl in my_tree_clades:
+        X.append(x_coords[cl])
+        Y.append(y_coords[cl])
+        text.append(cl.name)
+
+    df = pd.read_csv(metadata_file)
+
+    df.columns
```
Comment (sstripps1) [inline apps/dash-phylogeny/utils.py]: I think this line can be taken out

**Comment by sstripps1** [comment]
:dancer:

File: apps/dash-phylogeny/app.py (hunk 1 above)
Comment (ycaokris) [reply apps/dash-phylogeny/app.py]: Apparently no, removed

File: apps/dash-phylogeny/utils.py (hunk 2 above)
Comment (ycaokris) [reply apps/dash-phylogeny/utils.py]: > Screen Shot 2020-01-09 at 2 35 39 PM
> 
> Is it okay to have 2014.5 on the graph there?

Fixed by tickformat to integer 💯

**Comment by ycaokris** [comment]
All tests passed. Merging 🥇

**Comment by ycaokris** [comment]
@josegonzalez   I think I got a deployment error when merging dash-phylogeny from this branch to master. it passes and get deployed from this branch onto playground without any problem, merging ci job into master https://circleci.com/gh/plotly/dash-sample-apps/5842 has showed a path error/. Could you please give me some clues on how this error is raised? Thanks!

Screen Shot 2020-01-10 at 1 58 30 PM

**Pull request merged**